src/main/java/liquibase/ext/databricks/snapshot/jvm/ColumnSnapshotGeneratorDatabricks.java

```diff
@@ -43,4 +59,71 @@ protected DataType readDataType(CachedRow columnMetadataResultSet, Column column
         }
         return super.readDataType(columnMetadataResultSet, column, database);
     }
+
+    @Override
+    protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot snapshot) throws DatabaseException {
+        //This should work after fix on Databricks side
+        if (example instanceof Column) {
+            Column column = (Column) super.snapshotObject(example, snapshot);
+            //These two are used too often, avoiding them? otherwise there would be too much DB calls
+            if (!column.getRelation().getName().equalsIgnoreCase("databasechangelog")
+                    && !column.getRelation().getName().equalsIgnoreCase("databasechangeloglock")) {
```

Comment:
Even with skipping DBCL and DBCLL we are making additional call for every column. We have info for all columns in `show create table`.
column.getRelation() has all the columns, so we can iterate over them and add defaultValue information for all columns at once

Reply:
@SvampX as discussed, maybe DatabaseSnapshot.snapshotScratchPad could be useful to cache this information between executions.

Reply:
Moved show create table call to the Table snapshot generator. Storing create table statement in the `snapshotScratchPad` suggested by @filipelautert  with a show create table query as key. I believe such keys should be unique within a database as the query has full table name with catalog and schema.